src/parse.rs

```diff
@@ -799,13 +800,13 @@ fn resolve_attributes<'input>(
         let attr_name = ExpandedNameOwned { ns, name: attr.local.as_str() };
```

Comment:
It is somewhat pointless to possible allocate `ns` here only to throw it away again after deduplication, but I did not find the strength to shave this particular yak yet...